backends/rabbitmq-streams/write.go

```diff
@@ -0,0 +1,65 @@
+package rabbitmqStreams
+
+import (
+	"github.com/rabbitmq/rabbitmq-stream-go-client/pkg/stream"
+
+	"github.com/jhump/protoreflect/desc"
+	"github.com/pkg/errors"
+	"github.com/rabbitmq/rabbitmq-stream-go-client/pkg/amqp"
+	"github.com/sirupsen/logrus"
+
+	"github.com/batchcorp/plumber/cli"
+	"github.com/batchcorp/plumber/writer"
+)
+
+func Write(opts *cli.Options, md *desc.MessageDescriptor) error {
+
+	writeValues, err := writer.GenerateWriteValues(md, opts)
+	if err != nil {
+		return errors.Wrap(err, "unable to generate write value")
+	}
+
+	client, err := NewClient(opts)
+	if err != nil {
+		return err
+	}
+
+	defer client.Close()
+
+	producer, err := client.NewProducer(opts.RabbitMQStreams.Stream, &stream.ProducerOptions{
```

Comment:
I would use:
```golang
producer, err := env.NewProducer(streamName, 
		stream.NewProducerOptions().
		SetProducerName("xxxxx").SetBatchSize(xxxxx))
```

set `QueueSize` to 0 isn't a good choice. 
 I know I need to document all those parameters :)

Reply:
True, setting this to the number of messages I'm writing instead